android/build.gradle.kts

```diff
@@ -0,0 +1,21 @@
+allprojects {
```

Comment:
Same reason, we already have `build.gradle`.
Migration to Kotlin DSL  can be done in future. So please delete this